<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated with glade 3.22.2 -->
<interface>
  <requires lib="gtk+" version="3.16"/>
  <requires lib="libhandy" version="0.0"/>
  <!-- interface-css-provider-path style.css -->
  <template class="MainWindow" parent="GtkApplicationWindow">
    <property name="can_focus">False</property>
    <child type="titlebar">
      <object class="HdyTitleBar" id="handybar">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <child>
          <object class="HdyLeaflet" id="leaflet">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="mode_transition_type">slide</property>
            <property name="child_transition_type">slide</property>
            <signal name="notify::fold" handler="notify_fold_cb" after="yes" swapped="no"/>
            <signal name="notify::visible-child" handler="notify_header_visible_child_cb" swapped="no"/>
            <child>
              <object class="GtkHeaderBar" id="header">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="title">Conversations</property>
                <property name="show_close_button">True</property>
                <child>
                  <object class="GtkButton" id="add_button">
                    <property name="can_focus">True</property>
                    <property name="receives_default">True</property>
                    <child>
                      <object class="GtkImage">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="icon_name">list-add-symbolic</property>
                      </object>
                    </child>
                    <style>
                      <class name="image-button"/>
                    </style>
                  </object>
                </child>
                <child>
                  <object class="GtkToggleButton" id="search_button1">
                    <property name="can_focus">True</property>
                    <property name="receives_default">True</property>
                    <child>
                      <object class="GtkImage">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="icon_name">edit-find-symbolic</property>
                      </object>
                    </child>
                    <style>
                      <class name="image-button"/>
                    </style>
                  </object>
                  <packing>
                    <property name="position">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkMenuButton" id="menu_button">
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="focus_on_click">False</property>
                    <property name="receives_default">False</property>
                    <child>
                      <object class="GtkImage">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="icon_name">open-menu-symbolic</property>
                      </object>
                    </child>
                  </object>
                  <packing>
                    <property name="pack_type">end</property>
                    <property name="position">3</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="name">sidebar</property>
              </packing>
            </child>
            <child>
              <object class="GtkSeparator">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="orientation">vertical</property>
                <style>
                  <class name="sidebar"/>
                </style>
              </object>
            </child>
            <child>
              <object class="GtkHeaderBar" id="panel_headerbar">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="hexpand">True</property>
                <property name="show_close_button">True</property>
                <child>
                  <object class="GtkButton" id="back">
                    <property name="can_focus">False</property>
                    <property name="receives_default">False</property>
                    <property name="valign">center</property>
                    <property name="use_underline">True</property>
                    <child>
                      <object class="GtkImage">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="icon_name">go-previous-symbolic</property>
                        <property name="icon_size">1</property>
                      </object>
                    </child>
                    <style>
                      <class name="image-button"/>
                    </style>
                  </object>
                </child>
                <child>
                  <object class="GtkButton" id="group_button">
                    <property name="can_focus">False</property>
                    <property name="receives_default">False</property>
                    <property name="valign">center</property>
                    <property name="use_underline">True</property>
                    <child>
                      <object class="GtkImage">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="icon_name">system-users-symbolic</property>
                        <property name="icon_size">1</property>
                      </object>
                    </child>
                    <style>
                      <class name="image-button"/>
                    </style>
                  </object>
                  <packing>
                    <property name="pack_type">end</property>
                    <property name="position">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkButton" id="hangout_button">
                    <property name="can_focus">False</property>
                    <property name="receives_default">False</property>
                    <property name="valign">center</property>
                    <property name="use_underline">True</property>
                    <child>
                      <object class="GtkImage">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="icon_name">call-start-symbolic</property>
                        <property name="icon_size">1</property>
                      </object>
                    </child>
                    <style>
                      <class name="image-button"/>
                    </style>
                  </object>
                  <packing>
                    <property name="pack_type">end</property>
                    <property name="position">2</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="name">content</property>
              </packing>
            </child>
          </object>
        </child>
      </object>
    </child>
    <child>
      <object class="GtkStack" id="main_stack">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="hhomogeneous">False</property>
        <child>
          <object class="GtkBox" id="loading_state">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="orientation">vertical</property>
            <child>
              <object class="GtkSpinner" id="loading_spinner">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="active">True</property>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="name">loading_state</property>
            <property name="title" translatable="yes">page2</property>
          </packing>
        </child>
        <child>
          <object class="GtkBox" id="login_page">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="halign">center</property>
            <property name="valign">center</property>
            <property name="orientation">vertical</property>
            <property name="homogeneous">True</property>
            <child>
              <object class="GtkImage">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="resource">/com/dosch/HangoutsGTK/icons/placeholder.svg</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="login_button">
                <property name="label" translatable="yes">Log In</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="margin_top">29</property>
                <style>
                  <class name="suggested-action"/>
                </style>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">2</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="name">login_page</property>
            <property name="title" translatable="yes">page1</property>
            <property name="position">1</property>
          </packing>
        </child>
        <child>
          <object class="HdyLeaflet" id="content_box">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="mode_transition_type">slide</property>
            <property name="child_transition_type">slide</property>
            <child>
              <object class="GtkScrolledWindow" id="scrolled_window">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="vexpand">True</property>
                <property name="hscrollbar_policy">never</property>
                <child>
                  <object class="GtkViewport" id="conversation_list_viewport">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <child>
                      <placeholder/>
                    </child>
                  </object>
                </child>
              </object>
              <packing>
                <property name="name">content</property>
              </packing>
            </child>
            <child>
              <object class="GtkSeparator">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="orientation">vertical</property>
                <style>
                  <class name="sidebar"/>
                </style>
              </object>
            </child>
            <child>
              <object class="GtkStack" id="message_view">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="hexpand">True</property>
                <property name="vexpand">True</property>
                <property name="hhomogeneous">False</property>
                <property name="transition_type">crossfade</property>
                <child>
                  <object class="GtkBox">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="halign">center</property>
                    <property name="valign">center</property>
                    <property name="orientation">vertical</property>
                    <child>
                      <object class="GtkImage">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="resource">/com/dosch/HangoutsGTK/icons/placeholder.svg</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="label" translatable="yes">No group selected</property>
                        <style>
                          <class name="no_group_title"/>
                        </style>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="label" translatable="yes">select a conversation to start chatting</property>
                        <style>
                          <class name="no_group_description"/>
                        </style>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">2</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="name">default</property>
                    <property name="title" translatable="yes">page0</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="name">sidebar</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="name">content_box</property>
            <property name="title" translatable="yes">page0</property>
            <property name="position">2</property>
          </packing>
        </child>
      </object>
    </child>
  </template>
  <object class="GtkSizeGroup" id="end_pane_size_group">
    <widgets>
      <widget name="panel_headerbar"/>
      <widget name="message_view"/>
    </widgets>
  </object>
  <object class="HdyHeaderGroup" id="header_group">
    <headerbars>
      <headerbar name="header"/>
      <headerbar name="panel_headerbar"/>
    </headerbars>
  </object>
  <object class="GtkSizeGroup" id="start_pane_size_group">
    <widgets>
      <widget name="header"/>
      <widget name="scrolled_window"/>
    </widgets>
  </object>
  <object class="GtkAdjustment" id="column_linear_width_adjustment">
    <property name="upper">10000</property>
    <property name="value">500</property>
    <property name="step_increment">10</property>
    <property name="page_increment">100</property>
  </object>
  <object class="GtkAdjustment" id="column_width_adjustment">
    <property name="upper">10000</property>
    <property name="value">600</property>
    <property name="step_increment">10</property>
    <property name="page_increment">100</property>
  </object>
  <object class="GtkSizeGroup" id="separator_size_group"/>
</interface>
